src/main/kotlin/org/jitsi/utils/stats/BucketStats.kt

```diff
@@ -184,4 +206,4 @@ class Buckets(thresholdsNoMax: LongArray) {
 }
 
 /** Utility class with thresholds suitable for recording conference sizes. */
-class ConferenceSizeBuckets : BucketStats(longArrayOf(0, 1, 2, 5, 10, 20, 50, 100, 200, 300, 400, 500))
+class ConferenceSizeBuckets : BucketStats(listOf(0, 1, 2, 5, 10, 20, 50, 100, 200, 300, 400, 500))
```

Comment:
If you're removing the automatic MAX do you want to add it here?

Reply:
I did, in a later commit